### PR TITLE
chore(release): bump to v6.1.0 beta.1

### PR DESCRIPTION
**Pull request by shlokamin**
# Overview

Bump to 6.1.0 beta.1 

# Changelog

- Bump to 6.1.0-beta.1

# Review requests

- Check changelogs and make sure they seem reasonable. 

# Risk assessment

Low

### CODE REVIEW

**Review by sanni-t — APPROVED** [review]
🎉

File: app-shell/build/release-notes.md [hunk 1]
```diff
@@ -6,11 +6,9 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
 
 ---
 
-# Opentrons App Changes in 6.1.0-alpha.6
+# Opentrons App Changes in 6.1.0-beta.1
 
-Welcome to the v6.1.0 alpha release of the Opentrons App! This release focuses on adding support for the [Opentrons Heater-Shaker Module](https://shop.opentrons.com/heater-shaker-module/).
-
-<p align="center"><img alt="Isometric view of the Opentrons Heater-Shaker Module GEN1" src="https://s3.us-east-2.amazonaws.com/images.opentrons.com/modules/heater-shaker/hs_with_pcr_adapter.png" width="300" /></p>
+Welcome to the v6.1.0 beta release of the Opentrons App! This release focuses on adding support for the [Opentrons Heater-Shaker Module](https://shop.opentrons.com/heater-shaker-module/).
```
Comment (bndo) [inline app-shell/build/release-notes.md:11]: Did we lose the image on purpose?

Comment (shlokamin) [reply app-shell/build/release-notes.md:11]: yeah, it was an experiment we tried in the last alpha to get images to show up in the release notes that didnt work

Comment (sanni-t) [reply app-shell/build/release-notes.md:11]: ya, the markdown wasn't rendering, so we aren't putting it in this release

**Pull request merged**